services/venice-controller/src/main/java/com/linkedin/venice/controller/HelixAdminClient.java

```diff
@@ -96,6 +96,12 @@ void createVeniceStorageClusterResources(
       int numberOfPartition,
       int replicationFactor);
 
+  /**
+   * Check if a resource exists in a cluster by checking its ideal state.
+   * @return
```

Comment:
nit: We can remove empty return tag here.

Reply:
Done.